src/util/utilization.jl

```diff
@@ -0,0 +1,126 @@
+
+"""
+    `run_UTIL(data::Dict{String,<:Any})`
+
+Returns a priority restoration order for damaged components in a single network.
+
+# Example
+```
+julia> run_UTIL(data)
+Dict{String, Any} with 6 entries:
+    "4" => [("branch", "2")]
+    "1" => [("gen", "1")]
+    "5" => [("branch", "3")]
+    "2" => [("gen", "2")]
+    "6" => [("gen", "3")]
+    "3" => [("branch", "1")]
+```
+"""
+function run_UTIL(data::Dict{String,<:Any})
```

Comment:
I would recommend the name like `run_util` or `run_utilization`, my feeling at the moment is to prefer `run_utilization`.  Should be lower case, upper case is reserved for constant values following the JuMP style guide.